AdyenCard/Form/FormCardExpiryDateItemView.swift

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+/// A view representing a form card security code item.
+internal final class FormCardExpiryDateItemView: FormTextItemView<FormCardExpiryDateItem> {}
```

Comment:
why is this needed? the `FormCardExpiryDateItem` can just build a `FormTextItemView<FormCardExpiryDateItem>`, right?

Reply:
You are right, I saw that but also saw FormTextInputItemView and assumed we separate them for their own.

Reply:
`FormTextInputItemView` is also not needed :-D , I don't know why we have it.

Reply:
The only purpose of `FormTextInputItemView ` - it is a final non-generic class